tests/queries/tests.py

```diff
@@ -2079,6 +2083,18 @@ def test_distinct_ordered_sliced_subquery(self):
             ).order_by('id').values_list('id', flat=True), [2, 4]
         )
 
+    def test_logical_xor_subquery(self):
+        """Subquery should support logical XOR on supported databases"""
+        # Use subqueries to compute Q(name='first') ^ Q(id__lte=2)
+        self.assertQuerysetEqual(
+            NamedCategory.objects.filter(
+                Exists(NamedCategory.objects.filter(id=OuterRef('id'), name='first')) ^
+                Exists(NamedCategory.objects.filter(id=OuterRef('id'), id__lte=2))
+            ),
+            [2],
+            attrgetter('id'),
+        )
```

Comment:
I know that tests in `queries` are mixed up, however I would move it to the `Queries6Tests` class.

Reply:
Hm, ok. It seemed the most logical place to test the subquery was in `SubqueryTests`? 

I can't really tell what the intended goal of `Queries6Tests` is, but it doesn't matter much to me, so I can move it